themes/base/accordion.css

```diff
@@ -21,3 +21,14 @@
 	border-top: 0;
 	overflow: auto;
 }
+
+/* RTL support */
+.ui-accordion-rtl {
+	direction: rtl;
+}
+.ui-accordion-rtl .ui-accordion .ui-accordion-header {
+	padding: .5em .7em .5em .5em;
+}
+.ui-accordion-header-iconRtl.ui-icon {
+	transform: scaleX(-1);
```

Comment:
I very much disagree with this. This penalizes every user who uses RTL and a proper non-symmetrical icon. The documentation would have to tell users to intentionally build their icons backward or override our CSS.